app/bundles/PluginBundle/Controller/AjaxController.php

```diff
@@ -106,6 +106,77 @@ protected function getIntegrationLeadFieldsAction(Request $request)
     }
 
     /**
+     * Get the HTML for list of fields.
+     *
+     * @param Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\JsonResponse
+     */
+    protected function getIntegrationCompanyFieldsAction(Request $request)
+    {
+        $integration = $request->request->get('integration');
+        $settings    = $request->request->get('settings');
+
+        $dataArray = ['success' => 0];
+
+        if (!empty($integration) && !empty($settings)) {
+            /** @var \Mautic\PluginBundle\Helper\IntegrationHelper $helper */
+            $helper = $this->factory->getHelper('integration');
```

Comment:
This line should work with:
`$helper = $this->get('mautic.helper.integration')`
to avoid using factory.